requirements.txt

```diff
@@ -1,6 +1,5 @@
 lightgbm>=2.0.11
 numpy>=1.13.3
-scikit-image>=0.13.0
-scipy>=0.19.1
+opencv-python~=4.7.0.68
```

Comment:
I think it would be best to use `opencv-python-headless` to allow compilation on containers

Reply:
updated